modules/update-upgrading-cli.adoc

```diff
@@ -18,25 +18,11 @@ of the Customer Portal.
 
 * Install the OpenShift CLI (`oc`) that matches the version for your updated version.
 * Log in to the cluster as user with `cluster-admin` privileges.
-* Install the `jq` package.
+
 * Pause all `MachineHealthCheck` resources.
```

Comment:
May be we can handle this in another PR but we should give the steps to pause all `MachineHealthCheck` resources.

Reply:
Actually - the instructions are on the page: https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-cli.html#machine-health-checks-pausing_updating-cluster-cli (with the anchor at the end for specifics). I'll go ahead and delete this as a prereq.

Reply:
After more digging - looks like it's covered here: https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-cli.html#machine-health-checks-pausing_updating-cluster-cli so we should be set with the prereq as is

Reply:
I am not actually clear why we  recommend pausing MHCs before updating.  But I haven't had time to hunt around in  the pull request that originally added this.  I'd like to get the need clarified before we sink time into polishing the request, because we don't want to waste polishing time on something that's going away.

Reply:
@wking According to the page for pausing MHC's:

"During the upgrade process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the MachineHealthCheck resources before updating the cluster."

I suspect this is still relevant, but I could be wrong. Thoughts?

Reply:
Team decision to address MHC's in a new PR for the future. JIRA story here: https://issues.redhat.com/browse/OTA-873